ios/RNTPTDocumentView.m

```diff
@@ -2767,6 +2780,21 @@ - (BOOL)tabbedDocumentViewController:(PTTabbedDocumentViewController *)tabbedDoc
     return NO;
 }
 
+- (void)observeValueForKeyPath:(NSString *)keyPath ofObject:(id)object change:(NSDictionary *)change context:(void *)context
+{
+    if (context == TabChangedContext) {
+        if ([self.delegate respondsToSelector:@selector(tabChanged:currentTab:)]) {
+            PTDocumentTabItem *selectedItem = self.tabbedDocumentViewController.tabManager.selectedItem;
+            if (selectedItem != nil) {
+                NSURL *currentTab = selectedItem.documentURL ?: selectedItem.sourceURL;
+                [self.delegate tabChanged:self currentTab:[currentTab absoluteString]];
+            }
+        }
+    } else {
+        [super observeValueForKeyPath:keyPath ofObject:object change:change context:context];
```

Comment:
You should still call this even when you handle it. So you can call this blindly at the beginning of the method, then only do your special stuff if `context == TabChangedContext`.

Reply:
(Resolving as no longer necessary)